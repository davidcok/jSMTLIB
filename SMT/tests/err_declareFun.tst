; declare-fun with malformed arguments
(set-logic QF_UF)
(declare-fun x Bool)
(declare-fun x () A)
(declare-fun x (y z) Bool)
(declare-fun x ((y Bool)(y Bool)) Bool)
(declare-fun x () A A)

