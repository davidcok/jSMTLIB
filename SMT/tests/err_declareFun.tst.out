success
(declare-fun x Bool)
               ^^^^
(error "Expected a left parenthesis here, instead of a symbol")
(declare-fun x () A)
                  ^
(error "No such sort symbol declared: A")
(declare-fun x (y z) Bool)
                ^
(error "No such sort symbol declared: y")
(declare-fun x ((y Bool)(y Bool)) Bool)
                 ^
(error "No such sort symbol declared: y")
(declare-fun x () A A)
                    ^
(error "Too many arguments or extraneous material after the command or missing right parenthesis")
